selinux/services/udev.cil

```diff
@@ -0,0 +1,33 @@
+(type udev_exec_t)
+(call system_f (udev_exec_t))
+(context udev_exec_t (system_u object_r udev_exec_t (systemLow systemLow)))
+(filecon "/sbin/systemd-udevd" file udev_exec_t)
+(filecon "/usr/bin/udevadm" file udev_exec_t)
+(filecon "/sbin/udevadm" symlink udev_exec_t)
+
+; Do not reorder: label non-executable rules files as executable
+(type udev_rules_t)
+(call system_f (udev_rules_t))
+(filecon "/usr/lib/udev/rules.d(/.*)?" any (system_u object_r udev_rules_t (systemLow systemLow)))
+; TODO: separate label to be called by udev only?
```

Comment:
Or maybe better just allow udev executing itself and that's fine. No actual improvement in case we add more contexts.